lib/shared/global_config.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+FirebaseFirestore _firestoreInstance = FirebaseFirestore.instance;
+
+FirebaseFirestore getFirestoreInstance() {
+  return _firestoreInstance;
+}
+
+void setFirestoreInstance(FirebaseFirestore firestoreInstance) {
+  _firestoreInstance = firestoreInstance;
+}
```

Comment:
Is this the best way to initialise this?  

- We need to be able to set the firebase instance from tests, or else this will be largely untestable
- By doing this, we can set a real one when running the app, and a mock one when running unit tests
- Maybe this should be renamed to firebase_config
- I suppose this is okay for now.  I will likely have to do similar things during Authentication

Reply:
Renamed to firebase_config